youtube_dlc/extractor/niconico.py

```diff
@@ -546,11 +546,29 @@ def get_video_info_xml(items):
 
         webpage_url = get_video_info_web('watch_url') or url
 
+        # for channel movie and community movie
+        channel_id = (
+            try_get(api_data, lambda x: x['channel']['globalId'])
+            or try_get(api_data, lambda x: x['community']['globalId'])
+        )
```

Comment:
You can put this inside a single `try_get`. Same for `channel`

```suggestion
        channel_id = try_get(
            api_data,
            (lambda x: x['channel']['globalId'],
             lambda x: x['community']['globalId']))
```